### ISSUE TITLE
@date in docblock generates error in symfony

### ISSUE THREAD

**Issue by andriesvw** [IMSGlobal/LTI-Tool-Provider-Library-PHP#29]
I've installed the LTI-tool-provider under Symfony2. When e.g. ToolConsumer.php is accessed the following error is returned:

"[Semantical Error] The annotation \"@date\" in class IMSGlobal\\LTI\\ToolProvider\\ToolConsumer was never imported. Did you maybe forget to add a \"use\" statement for this annotation?"

**Comment by renaatdemuynck** [comment]
This is correct. `@date` is not a valid PHPDoc tag. It should either be removed or replaced by `@since`.
